tables/afr-za-g1.ctb

```diff
@@ -15,10 +15,11 @@
 # accurate nor complete. It hasn't been verified by the table
 # author yet. It is merely an attempt by the liblouis maintainers
 # to get some sensible initial values in place.
-# ------------
 #
-# Based on World Braille Usage 2013
-# <http://liblouis.org/braille-specs/world-braille-usage-third-edition.pdf>.
+# TODO: Please add a reference to official documentation about
+# the implemented braille code. Preferably submit the documents
+# to https://github.com/liblouis/braille-specs.
+# ------------
 #
 # Copyright (C) 2014, Greg Kearney <gkearney@gmail.com>
```

Comment:
Does this need to be updated?

Reply:
And instead adding this TODO it would of course be much better to actually add a link, or to say that there is no documentation.

Reply:
Date was fixed.